test-configs.yaml

```diff
@@ -1328,22 +1334,32 @@ test_configs:
   - device_type: qcom-qdf2400
     test_plans: [boot]
 
-  - device_type: qemu_arm
+  - device_type: qemu-arm-virt-gicv2
     test_plans:
       - boot_qemu
       - simple_qemu
-      - uefi-arm-gicv2
-      - uefi-arm-gicv3
+      - uefi-arm
       - v4l2_compliance_qemu_vivid
 
-  - device_type: qemu_arm64
+  - device_type: qemu-arm-virt-gicv3
     test_plans:
       - boot_qemu
       - simple_qemu
-      - uefi-arm64-gicv2
-      - uefi-arm64-gicv3
+      - uefi-arm
       - v4l2_compliance_qemu_vivid
 
+  - device_type: qemu-arm64-virt-gicv2
+    test_plans:
+      - boot_qemu
+      - uefi-arm64
+      - simple_qemu
+
+  - device_type: qemu-arm64-virt-gicv3
+    test_plans:
+      - boot_qemu
+      - uefi-arm64
```

Comment:
Alphabetical order problem

Reply:
I dont see it

Reply:
`simple` comes before `uefi`

Reply:
the UI show me only the 4 last line... but the mail show me the whole diff

Reply:
Surely you have a local copy of the file anyway?